src/Microsoft.Extensions.CodeGeneration.Utils/Workspaces/ProjectJsonWorkspace.cs

```diff
@@ -214,7 +212,11 @@ private static CSharpCompilationOptions AddSigningOptions(CSharpCompilationOptio
             {
                 keyFile = Path.GetFullPath(Path.Combine(projectDirectory, compilerOptions.KeyFile));
 
-                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || useOssSigning)
+                if (
+#if !NET451                    
```

Comment:
Currently scaffolding is supported only on windows through VS. 
Also this is code copied from ProjectModel.Workspace. 

Once that targets net451, this code will be removed altogether.